packages/components/src/snackbar/style.scss

```diff
@@ -34,7 +34,7 @@
 	flex-shrink: 0;
 	line-height: $default-line-height;
 
-	&:not(:disabled):not([aria-disabled="true"]):not(.is-default) {
+	&:not(:disabled):not([aria-disabled="true"]):not(.is-default):not(:focus) {
```

Comment:
Instead of changing this selector which is already very long, you can simply add `background-color: transparent;` to the ruleset:

```
	&:not(:disabled):not([aria-disabled="true"]):not(.is-default) {
		text-decoration: underline;
		background-color: transparent;

		&:hover {
			color: $white;
			text-decoration: none;
		}
	}
```

Reply:
Here's even better code that cleans things up a bit:

```
.components-snackbar__action.components-button {
	margin-left: $grid-size * 4;
	color: $white;
	height: auto;
	flex-shrink: 0;
	line-height: $default-line-height;
	padding: 0;

	&:not(:disabled):not([aria-disabled="true"]):not(.is-default) {
		text-decoration: underline;
		background-color: transparent;

		&:hover {
			color: $blue-medium-400;
		}

		&:focus {
			color: $white;
			box-shadow: none;
			outline: 1px dotted $white;
		}
	}
}
```

Neutral:

![Screenshot 2019-10-14 at 12 55 50](https://user-images.githubusercontent.com/1204802/66746425-18c16f00-ee82-11e9-95f4-9a063df533d3.png)

Hover:

![Screenshot 2019-10-14 at 12 56 01](https://user-images.githubusercontent.com/1204802/66746437-20811380-ee82-11e9-8f1b-a15de00b9ce8.png)

_Yes, that light blue is 4.9:1 contrast_

Focus:

![Screenshot 2019-10-14 at 12 56 20](https://user-images.githubusercontent.com/1204802/66746455-28d94e80-ee82-11e9-9aef-6642fba41f6c.png)

Reply:
We want to include this PR in the release today, so I applied the code with 075ccfd. I will test locally again and merge if it works for me. Major props @jasmussen for your help.